mibidata/tests/test_pseudodepths.py

```
@@ -0,0 +1,149 @@
+        return data
+
+    def test_split_into_one(self):
+        """If we split into one output file, we should get the same file out.
+        """
+        cycles_per_pixel, cycles_per_scan = pseudodepths.divide(
+            self.msdf, 1, self.tempdir)
+        self.assertEqual(cycles_per_pixel, 10)
+        self.assertEqual(cycles_per_scan, 10)
+        new_file = os.path.join(self.tempdir, 'Depth0', 'Image.msdf')
+        self.assertTrue(os.path.exists(new_file))
+        with open(self.msdf, 'rb') as infile:
+            expected_buffer = infile.read()
+        with open(new_file, 'rb') as infile:
+            new_buffer = infile.read()
+        self.assertEqual(new_buffer, expected_buffer)
+
+    def test_split_into_two(self):
+        cycles_per_pixel, cycles_per_scan = pseudodepths.divide(
+            self.msdf, 2, self.tempdir)
+        self.assertEqual(cycles_per_pixel, 10)
+        self.assertEqual(cycles_per_scan, 5)
+        depth0 = os.path.join(self.tempdir, 'Depth0', 'Image.msdf')
+        depth1 = os.path.join(self.tempdir, 'Depth1', 'Image.msdf')
+        with open(depth0, 'rb') as infile:
+            depth0_header = infile.read(pseudodepths.HEADER_SIZE)
+            depth0_sat = infile.read(NUM_PIXELS * pseudodepths.SAT_ENTRY_SIZE)
+            depth0_data = infile.read()
+        with open(depth1, 'rb') as infile:
+            depth1_header = infile.read(pseudodepths.HEADER_SIZE)
+            depth1_sat = infile.read(NUM_PIXELS * pseudodepths.SAT_ENTRY_SIZE)
+            depth1_data = infile.read()
+
+        self.assertEqual(depth0_header, self.header)
+        self.assertEqual(depth1_header, self.header)
+        self.assertEqual(depth0_sat, self._pack_sat(DEPTH0))
+        self.assertEqual(depth1_sat, self._pack_sat(DEPTH1))
+        self.assertEqual(depth0_data, self._pack_data(DEPTH0))
+        self.assertEqual(depth1_data, self._pack_data(DEPTH1))
+
+    def test_split_into_three(self):
+        with self.assertRaises(ValueError):
```

Some explanation here would be great, such as: This should throw an error because the number of cycles is not divisible by the number of desired splits.